src/olympia/pages/templates/pages/faq.html

```diff
@@ -95,38 +90,19 @@ <h3 id="using-addons">{{ _('Using Add-ons') }}</h3>
         <a href="{{ troubleshooting_url }}">Troubleshooting Extensions and
         Themes</a>.{% endtrans %}</dd>
 
-    {% if APP == amo.FIREFOX or APP == amo.MOBILE %}
-        <dt>{{ _('How do I install add-ons without restarting Firefox?') }}</dt>
-        <dd>{% trans sdk_url="https://developer.mozilla.org/en-US/Add-ons/SDK",
-               bootstrap_url="https://developer.mozilla.org/Extensions/Bootstrapped_extensions" %}
-            In Firefox, add-ons marked with "No restart required" can be installed
-            without restarting. These add-ons have been created using the
-            <a href="{{ sdk_url }}">Add-on SDK</a> or <a href="{{ bootstrap_url }}">bootstrapping</a>.
-            Other add-ons will still require a restart before you can use them.
-            {% endtrans %}</dd>
-    {% endif %}
-
     <dt>{{ _('How do I keep add-ons up-to-date?') }}</dt>
-    <dd>{% trans plugincheck_url="http://www.mozilla.com/plugincheck/",
-           plugin_url="http://support.mozilla.com/kb/Using+plugins+with+Firefox"  %}
-        Add-ons, unlike plugins, are automatically checked for updates once
-        every day. In Firefox, updates are automatically installed by
-        default. Versions of Firefox prior to 4 (and other applications) will alert
-        you that updates to your add-ons are available.
-        <a href="{{ plugin_url }}">Plugins</a> are not currently automatically
-        checked for updates, so be sure to regularly visit the
-        <a href="{{ plugincheck_url }}">Plugin Check</a> page to stay
-        up-to-date.{% endtrans %}</dd>
+    <dd>{{ _('Add-ons are automatically checked for updates once every day. In '
+             'Firefox, updates are automatically installed by default. ') }}
+    </dd>
 
     <dt>{{ _('Are add-ons safe to install?') }}</dt>
-    <dd>{% trans learnmore_url="https://addons.mozilla.org/developers/docs/policies/reviews" %}
-        Unless clearly marked otherwise, add-ons available from this gallery
-        have been checked and approved by Mozilla's team of editors and are
-        safe to install. We recommend that you only install approved add-ons.
-        If you wish to install unapproved add-ons or add-ons from third-party
-        websites, use caution as these add-ons may harm your computer or
-        violate your privacy. <a href="{{ learnmore_url }}">Learn more about
-        our approval process</a>{% endtrans %}</dd>
+    <dd>{% trans learnmore_url="https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews" %}
+        Add-ons available from this gallery are regularly reviewed by Mozilla's
+        team of reviewers and should be safe to install. If you wish to install
+        add-ons from third-party websites, use caution as these add-ons may harm
+        your computer or violate your privacy.
+        <a href="{{ learnmore_url }}">Learn more about our approval
```

Comment:
"next rewrite"? Should we merge this anyway?

@wagnerand since you already looked at this, might giving this a final review? I couldn't spot anything so far but I guess you can say a bit more about it content-wise.